zerver/actions/user_groups.py

```diff
@@ -158,6 +158,8 @@ def promote_new_full_members() -> None:
 def do_send_create_user_group_event(
     user_group: UserGroup, members: List[UserProfile], direct_subgroups: Sequence[UserGroup] = []
 ) -> None:
+    can_mention_group = user_group.can_mention_groups.first()
```

Comment:
This is kinda workaround for now assuming that the `can_mention_groups` is set to a single group in this commit and we will add support to set the setting to multiple groups through API in further commits.

Reply:
Makes sense!